query/src/pipelines/transforms/group_by/aggregator_polymorphic_keys.rs

```diff
@@ -214,6 +200,93 @@ impl PolymorphicKeysHelper<HashMethodKeysU64> for HashMethodKeysU64 {
     }
 }
 
+impl PolymorphicKeysHelper<HashMethodKeysU128> for HashMethodKeysU128 {
```

Comment:
I think it might be better to apply macro for `u128`, `u256` and `u512` here. cc@zhang2014